.gitignore

```diff
@@ -10,6 +10,34 @@
 /setup.h
 /*tags
 /autom4te.cache
+/.deps
+/.pch
+/version-script
+/bk-deps
+/bk-make-pch
+/change-install-names
+
+# ignore all Linux GTK+ 3 compile objects
+/advdll*.o
+/auidll*.o
+/basedll*.o
+/coredll*.o
+/gldll*.o
+/htmldll*.o
+/mediadll*.o
+/netdll*.o
+/propgriddll*.o
+/qadll*.o
+/ribbondll*.o
+/richtextdll*.o
+/sound*.o
+/stcdll*.o
+/utils/wxrc/wxrc*.o
+/webviewdll*.o
+/wxregex*.o
+/wxscintilla*.o
+/xmldll*.o
+/xrcdll*.o
```

Comment:
I'm confused about how you have any of these `*.o` files in your root tree... how are you building wxWidgets? are you running `make install`? what are you setting for your prefix?

Pretty much all official documentation recommends building wxWidgets in a subfolder, i.e.:
```
$ mkdir gtk3-static-debug
$ cd gtk3-static-debug/
$ ../configure --disable-shared --enable-debug
$ make
```

Pay special attention to the double periods prepended to the `configure` step. The build won't create or touch any files outside of your new build directory, which is often recommended for your local `.gitignore`.